podcasts/Onboarding/Import/ImportDetailsView.swift

```diff
@@ -1,7 +1,17 @@
 import SwiftUI
 
 struct ImportDetailsView: View {
+    enum OPMLImportResult {
```

Comment:
Wdyt about renaming that to `OPMLURLImportResult`? Just so we make sure to convey that this is for external URLs.

Reply:
updated